docs/shell-tips.md

```diff
@@ -1,83 +1,64 @@
 # Shell tips
 
-The *shell* (sometimes called *terminal* or *console*), is a **command line
-interpreter**. This is how most shells look like:
+The *shell* a is a **command line interpreter**. To use it you can open a
+*terminal* (sometimes called a *console*). This is how most terminal windows
+look like:
 
 ![An example shell window](shell-screenshot.png)
 
 If you haven't used it before, you should probably take a look at
-[this tutorial](http://linuxcommand.org/lc3_learning_the_shell.php). If you're
+[this tutorial](http://linuxcommand.org/lc3_learning_the_shell.php).
+
+If you're
 using Windows,
 [these videos](https://www.youtube.com/playlist?list=PL6gx4Cwl9DGDV6SnbINlVUd0o2xT4JbMu)
 may be useful too, but keep in mind that the following tips only apply to
-Linux/macOS environments (Unix-like shells).
+Linux/macOS environments (Unix shells). You can also use a Unix-like shell
+on Windows, for example [Cygwin](https://www.cygwin.com/).
 
 ## The prompt (`$`)
 
-When searching in Google, or in Zulip's docs, you'll find commands that have
-at the beginning a dollar sign `$`, or a dollar sign preceded by some text
+When searching Google, or Zulip's docs, you'll find commands that begin
+with a dollar sign `$` or a dollar sign preceded by some text
 (e.g. `(zulip-venv)vagrant@vagrant-base-trusty-amd64:~$`).
 
 This is called the **prompt**, and it's only an indicator that the shell is
-awaiting new orders. You shouldn't type that, since it isn't part of the
-commands.
-
-## Running commands as root (`sudo`)
+awaiting new orders. The prompt can contain useful information, let's look
+at `(zulip-venv)vagrant@vagrant-base-trusty-amd64:~$`:
```

Comment:
Agreed, though I don't think you'd have `(zulip-venv)` on your machine, it's more likely to just be a `venv`.